### PR TITLE
Add Attendance List into SCA

### PR DESCRIPTION
**Pull request by FlyingDonkeys**
- will be incorporating attendance into the sca

### CODE REVIEW

**Comment by FlyingDonkeys** [comment]
- Overloaded the constructor for SCA so as to distinguish between creation of new SCA object and updating SCA object
- Wrote test cases for new SCA

File: src/main/java/tahub/contacts/model/studentcourseassociation/StudentCourseAssociation.java
Comment (Zeepheru) [inline src/main/java/tahub/contacts/model/studentcourseassociation/StudentCourseAssociation.java:1]: Good idea to overload to differentiate constructors for creating a new SCA and loading an SCA from storage 👍

**Review by Zeepheru — APPROVED** [review]
LGTM

**Comment by FlyingDonkeys** [comment]
Merge updated SCA with Attendance

**Pull request closed without merging**